### PR TITLE
Update labeler docs on how to create a labeler, update models, etc.

### PR DESCRIPTION
**Pull request by Eilon**
Some of the existing docs were re-organized, and many new docs were added.

- The main readme has basic info
- The Documentation folder has a readme for how to set things up, and a secondary file with some implementation details

Some of the docs are still incomplete, but it's far more extensive than we had.

### CODE REVIEW

File: Documentation/README.md [hunk 1]
```diff
@@ -1,64 +1,181 @@
...
+1. Use an existing labeler app that supports the same GitHub org and add your repo's data to it
+1. Create a new labeler app and add your repo's data to it
+
+Please note that a single labeler app can support only one org, so if your org isn't supported yet, you must create a new labeler app instance.
+
+If your are adding your repo to an existing labeler, see the [Web App Service List](#web-app-service-list) for a list of existing apps.
+
+Follow these steps to create a new labeler app:
+
+1. To create a new labeler, you must have access to the `GitHubIssueLabeller` resource group in the `DDFun IaaS Dev Shared Public` Azure Subscription. Contact the DevDiv Azure Ops email alias to request access.
```
Comment (dakersnar) [inline Documentation/README.md:20]: Is there an easy fix to the spelling of `GitHubIssueLabeller`? If not, what's the long-term solution here?

**Review by dakersnar — APPROVED** [review]
This looks good, thank you! I'll use this document to update the dotnet/runtime model as soon as you give me the "ok", and I can provide further feedback if I run into any snags.

File: Documentation/README.md (hunk 1 above)
Comment (Eilon) [reply Documentation/README.md:20]: Unfortunately it does not seem possible to rename an Azure Resource Group at this time, so it's stuck. We could create a new group and move everything to it, but that would likely break various things. I think the long-term solution is that we learn as best we can to live with this mistake 😄

**Comment by dakersnar** [comment]
Sounds good, I will schedule a meeting when I find some time. Probably later this week or next week.

**Pull request merged**